akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/LogCapturing.scala

```diff
@@ -49,6 +49,22 @@ trait LogCapturing extends BeforeAndAfterAll { self: TestSuite =>
     }
   }
 
+  /**
+   * If true, the logs will be cleared after each test.
+   *
+   * This is not enabled by default for Scala tests, since state may be shared between
+   * sequential tests, and logs from earlier tests may be important.
+   */
+  protected val clearLogsAfterEachTest: Boolean = false
```

Comment:
A def is probably better for flexibility because it can still be overridden with a val, but a val can not be overriden by a def.